### ISSUE TITLE
[CLI] php app/console debug:event-dispatcher doesn't work

### ISSUE THREAD

**Issue by craigh** [zikula/core#2507]
```shell
cli% php app/console debug:event-dispatcher
[event_dispatcher] Registered listeners by event
                                                                              
  [Symfony\Component\DependencyInjection\Exception\InvalidArgumentException]  
  The service definition "request" does not exist.                            

debug:event-dispatcher [--format FORMAT] [--raw] [--] [<event>]
```

```shell
cli% php app/console debug:event-dispatcher -vvv
[event_dispatcher] Registered listeners by event
                                                                
  [Symfony\Component\DependencyInjection\Exception\InvalidArgumentException]  
  The service definition "request" does not exist.                            

Exception trace:
 () at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:868
 Symfony\Component\DependencyInjection\ContainerBuilder->getDefinition() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:478
 Symfony\Component\DependencyInjection\ContainerBuilder->get() at /Applications/MAMP/htdocs/core.git/src/lib/legacy/Zikula/ServiceManager.php:117
 Zikula_ServiceManager->get() at /Applications/MAMP/htdocs/core.git/src/lib/util/ServiceUtil.php:95
 ServiceUtil::get() at /Applications/MAMP/htdocs/core.git/src/lib/legacy/Zikula/AbstractBase.php:135
 Zikula_AbstractBase->__construct() at /Applications/MAMP/htdocs/core.git/src/app/cache/dev/appDevDebugProjectContainer.php:5260
 appDevDebugProjectContainer->getZikularoutesmodule_ControllerHelperService() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Container.php:327
 Symfony\Component\DependencyInjection\Container->get() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:469
 Symfony\Component\DependencyInjection\ContainerBuilder->get() at /Applications/MAMP/htdocs/core.git/src/lib/legacy/Zikula/ServiceManager.php:117
 Zikula_ServiceManager->get() at /Applications/MAMP/htdocs/core.git/src/app/cache/dev/appDevDebugProjectContainer.php:5104
 appDevDebugProjectContainer->getZikula_ExceptionListenerService() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Container.php:327
 Symfony\Component\DependencyInjection\Container->get() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:469
 Symfony\Component\DependencyInjection\ContainerBuilder->get() at /Applications/MAMP/htdocs/core.git/src/lib/legacy/Zikula/ServiceManager.php:117
 Zikula_ServiceManager->get() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/ContainerAwareEventDispatcher.php:188
 Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher->lazyLoad() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/ContainerAwareEventDispatcher.php:125
 Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher->getListeners() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:99
 Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->getListeners() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php:339
 Symfony\Bundle\FrameworkBundle\Console\Descriptor\TextDescriptor->describeEventDispatcherListeners() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php:73
 Symfony\Bundle\FrameworkBundle\Console\Descriptor\Descriptor->describe() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/Console/Helper/DescriptorHelper.php:71
 Symfony\Component\Console\Helper\DescriptorHelper->describe() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php:86
 Symfony\Bundle\FrameworkBundle\Command\EventDispatcherDebugCommand->execute() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:886
 Symfony\Component\Console\Application->doRunCommand() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:195
 Symfony\Component\Console\Application->doRun() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at /Applications/MAMP/htdocs/core.git/src/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:126
 Symfony\Component\Console\Application->run() at /Applications/MAMP/htdocs/core.git/src/app/console:24


debug:event-dispatcher [--format FORMAT] [--raw] [--] [<event>]
```

**Comment by craigh** [comment]
this can be closed when #2576 is merged.

**Comment by craigh** [comment]
closed by #2576
